client/extensions/woocommerce/app/settings/shipping/index.js

```diff
@@ -41,6 +41,7 @@ class Shipping extends Component {
 	};
 
 	render = () => {
+		alert( 'entra' )
```

Comment:
Do we need this? :D

Reply:
Spanish lessons right there! :sweat_smile: